sentry-android-fragment/src/main/java/io/sentry/android/fragment/SentryFragmentLifecycleCallbacks.kt

```diff
@@ -9,13 +9,30 @@ import androidx.fragment.app.FragmentManager.FragmentLifecycleCallbacks
 import io.sentry.Breadcrumb
 import io.sentry.HubAdapter
 import io.sentry.IHub
+import io.sentry.ISpan
 import io.sentry.SentryLevel.INFO
+import io.sentry.SpanStatus
 
 @Suppress("TooManyFunctions")
 class SentryFragmentLifecycleCallbacks(
-    private val hub: IHub = HubAdapter.getInstance()
+    private val hub: IHub = HubAdapter.getInstance(),
```

Comment:
`HubAdapter` is just a wrapper around the `Sentry` static class, to make it testable, it's going to use the active `hub` in the Sentry class, or am I missing something?

Reply:
The problem is that when you do:

```
IHub hub = HubAdapter.getInstance();
Sentry.init();

hub.capture() -> This won't work because. 

But if you do `HubAdapter.getInstance().capture` that would work.

So the idea is to always do `HubAdapter.getInstance()` (through a local `Callable<IHub>`)

Reply:
Fair, the Hub instance is always passing the call through to the static class so changes to the Hub instnace bound to that won't take effect here.